src/stratigraphy/metadata/metadata.py

```
@@ -99,14 +99,23 @@ def __str__(self) -> str:
 
 @dataclass
 class BoreholeMetadataList(metaclass=abc.ABCMeta):
-    """Metadata for stratigraphy data."""
+    """Metadata for stratigraphy data. This class is a list of BoreholeMetadata objects. Each object corresponds to a
+    single file."""
```

Ruff

1 blank line required between summary line and description

(D205)